src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
+
+import "./NavSearchResult.scss";
+
+interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
+}
+
+export const NavSearchResult: React.FC<NavSearchResultProps> = ({
+  title,
+  description,
+  image,
+  isAvatar,
+  onClick,
+}) => {
+  return (
+    <div className="nav-dropdown--search-result" onClick={onClick}>
+      <div
+        className={classNames("nav-dropdown--search-result-image", {
+          "nav-dropdown--search-result-avatar": isAvatar,
+        })}
```

Comment:
Extract these into a `const imageClasses = classNames(` rather than inlined here.

Reply:
don't need to build classNames here anymore: https://github.com/sparkletown/sparkle/pull/1328/commits/eb24607d724c23e94c3d4d1a40d7df37f6d9489b